README.md

```diff
@@ -212,6 +213,49 @@ is set to `true`.
 See html5rocks's [article](http://www.html5rocks.com/en/tutorials/cors/)
 for more info.
 
+##### useCueTags
+* Type: `boolean`
+* can be used as an initialization option
+
+When the `useCueTags` property is set to `true,` a text track is created with
+label 'hls-segment-metadata' and kind 'metadata'. The track is then added to
+`player.textTracks()`. Whenever a segment associated with a cue tag is playing,
+the cue tags will be listed as a properties inside of a stringified JSON object
+under its active cue's `text` property. The properties that are currently
+supported are cueOut, cueOutCont, and cueIn. Changes in active cue may be
+tracked by following the Video.js cue points API for text tracks. For example:
+
+```javascript
+var
+  textTracks = player.textTracks(),
+  cuesTrack,
+  i;
+
+for (i = 0; i < textTracks.length; i++) {
+  if (textTracks[i].label === 'hls-segment-metadata') {
+    cuesTrack = textTracks[i];
+  }
+}
+
+cuesTrack.oncuechange = function() {
+  var
```

Comment:
I've been starting to use es6 in code samples as well when it's reasonable. So, these would just end up being `let`s. But no strong preference either way.